org.contourterminal.Contour.yml

```diff
@@ -0,0 +1,91 @@
+app-id: org.contourterminal.Contour
+name: Contour Terminal Emulator
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: contour
+finish-args:
+  - --filesystem=host
```

Comment:
Why does it need host access if it launches shell on the host with flatpak-spawn?

Reply:
This is a left-over from back in the day, when I didn't know I will be needing flatpak-spawn to create a non-sandboxed process. I've dropped that in a new commit (but didn't test, as I'm on mobile / commute) just yet. :)